tests/containers/bci_logs.pm

```diff
@@ -0,0 +1,62 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: bci-tests log parser
+#
+#   This module is used to parse the result files generated by BCI-tests
+#   tox commands.
+#   BCI-tests produce separate XUnit results files for each environment.
+#   This merges all files together into a single xml file that will
+#   be used by OpenQA to represent the results in "External results"
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'opensusebasetest';
+use XML::LibXML;
+use testapi;
+use File::Basename;
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    assert_script_run("cd /root/BCI-tests");
+    my $dom = XML::LibXML::Document->new('1.0', 'utf-8');
+    my $root = $dom->createElement('testsuites');
+    $dom->setDocumentElement($root);
+    my $result_files = script_output('ls | grep junit');
```

Comment:
```suggestion
    my $result_files = script_output('find /root/BCI-tests -type f -name "*.junit"');
```
With this, you can also avoid `cd`.

Reply:
I am renaming those files after bci_test.pm module, but the suggestion is still valid, thank you.

Reply:
Updated with both suggestions. `find /root/BCI-tests -type f -name "junit_*.xml"`